tests/unit/components/playback/controls/EditCustomer.spec.js

```diff
@@ -0,0 +1,514 @@
+import { mount } from "@vue/test-utils";
+import ComponentToTest from "@/components/playback/controls/player/EditCustomer.vue";
+import { EditCustomer as DistComponentToTest } from "@/dist/mantarray.common";
+
+import Vue from "vue";
+import Vuex from "vuex";
+import { createLocalVue } from "@vue/test-utils";
+import BootstrapVue from "bootstrap-vue";
+import "bootstrap/dist/css/bootstrap.min.css";
+import uuid from "@tofandel/uuid-base62";
+import { TextValidation } from "@/js_utils/text_validation.js";
+let wrapper = null;
+
+const localVue = createLocalVue();
+localVue.use(Vuex);
+localVue.use(BootstrapVue);
+localVue.use(uuid);
+
+let NuxtStore;
+let store;
+
+describe("EditCustomer.vue", () => {
+  const editcustomer = {
+    uuid: "",
+    apikey: "",
+    nickname: "",
+  };
+  beforeEach(async () => {
+    const propsData = {
+      dialogdata: editcustomer,
+      dataindex: 0,
+    };
+    wrapper = mount(ComponentToTest, {
+      propsData,
+      store,
+      localVue,
+    });
+  });
+  test("When mounting EditCustomer from the build dist file, Then it loads successfully and the `Edit Customer` defined title text is rendered", () => {
+    const propsData = {
+      dialogdata: editcustomer,
+      dataindex: 0,
+    };
+    wrapper = mount(DistComponentToTest, {
+      propsData,
+      store,
+      localVue,
+    });
+
+    const target_span = wrapper.find(
+      ".span__editcustomer-form-controls-content-title"
+    );
+
+    expect(target_span.text()).toStrictEqual("Edit Customer Account ID");
+  });
+});
+
+describe("EditCustomer.enter_uuidbase57", () => {
+  const uuid_base57 = "2VSckkBYr2An3dqHEyfRRE";
+
+  beforeEach(async () => {
+    store = await NuxtStore.createStore();
```

Comment:
this component does not use Vuex does it? if it is independent of Vuex, then there is no need to include `store` when mounting or have all of these `beforeAll` `beforeEach` etc. related to creating a Vuex store.  Please check all 4 spec files to see if this applies to them also

Reply:
Agreed and removed.